File: 6-kyu/Find The Divisors/FindDivisors.py
# Kata can be found at: https://www.codewars.com/kata/544aed4c4a30184e960010f4

def divisors(integer):
    
    list = [];
    i = 2;
    while i < integer:
        if integer % i == 0:
            list.append(i)
        i = i + 1

    if not list:
        return str(integer) + ' is prime'
    return list

File: 6-kyu/Find The Divisors/FindDivisors.js
// Kata can be found at: https://www.codewars.com/kata/544aed4c4a30184e960010f4

function divisors(integer) {
  
  var arr = [];
  
  for(var i = 2; i < integer; i++)  {
  
    if(integer % i == 0)  arr.push(i);
    
  }
  
  return arr.length == 0 ? integer + " is prime" : arr;
  
};
